packages/clipboard/readme.md

```diff
@@ -1,3 +1,54 @@
-# Clipboard
+# @creately/clipboard
 
-Clipboard Service which uses system clipboard as primary and local storage as fallback.
+> Clipboard service which uses system clipboard as primary and local storage as fallback.
+
+Support:  All browsers and Embedded( iframe ).
+
+## Install
+
+```
+$ npm install @creately/clipboard
+```
+
+## Usage
+
+```js
+import { Clipboard } from '@creately/clipboard';
+
+new Clipboard().copy('your data');
+
+new Clipboard().paste();
+//=> 'your data'
+```
+
+If required you can use the provider to use the same instance.
+```js
+// Add this in your module.
+{ provide: Clipboard, useFactory: () => new Clipboard() },
+
+// Add this in constructor.
+constructor( protected clipboard: Clipboard ) {}
```

Comment:
Can you give a full class instead of parts separately.

```ts
class MyClass {
  constructor( private clipboard: Clipboard ) {}

  // ...
}
```